widgetssdk/src/main/java/com/glia/widgets/engagement/domain/FlipVisitorCameraUseCase.kt

```diff
@@ -44,6 +46,8 @@ internal class FlipVisitorCameraUseCaseImpl(private val repository: EngagementRe
      *         If the current camera is the last in the list, it will return the first camera device.
      */
     private fun getNextAvailableCamera(cameras: List<CameraDevice>, currentVisitorCamera: CameraDevice?): CameraDevice? {
+        Logger.w(TAG, "Getting the next available camera instead of the following camera according to the facing")
+
```

Comment:
@AndriiHorishniiMOC Please add this log to the [doc](https://docs.google.com/spreadsheets/d/1ty9bhrbX3VTXBThI5f4GFzy5X-RbEHOCPWfXd7nGzbo/edit#gid=0). Also, please ask iOS team if they want to add the same log. 😉